### PR TITLE
feat: grid helpers

### PR DESCRIPTION
**Pull request by adith-a-danthi**
Added Grid Helper classes
- Grid Template Columns
- Grid Template Rows
- Grid Utility classes

image

### CODE REVIEW

File: css/docs.css [hunk 1]
```diff
@@ -107,6 +108,24 @@
   color: var(--color-gray-dark);
 }
 
+.grid-item {
+  display: grid;
+  place-content: center;
+  padding: 1rem;
+  border-radius: 5px;
+  background-color: var(--color-default);
+  color: white;
```
Comment (stutisk) [inline css/docs.css:117]: Can use color as a variable

File: css/docs.css [hunk 2]
```diff
@@ -107,6 +108,24 @@
...
+  font-size: 1.5rem;
```
Comment (stutisk) [inline css/docs.css:118]: Font size too

**Review by stutisk — COMMENTED** [review]
Else code looks good to me

File: css/docs.css (hunk 2 above)
Comment (adith-a-danthi) [reply css/docs.css:118]: These styles are specific to the documentation site, so didn't use variables.

**Review by ArjunGTX — APPROVED** [review]
Looks like you have seperate classes for each grid property. Nice work btw

**Pull request merged**